pkg/ctl/cmdutils/cmdutils.go

```diff
@@ -67,6 +68,14 @@ func LogPlanModeWarning(plan bool) {
 	}
 }
 
+// LogRegionAndVersionInfo will log the selected region and build version
+func LogRegionAndVersionInfo(meta *api.ClusterMeta) {
+	if meta != nil {
+		logger.Info("eksctl version %s", version.GetVersion())
```

Comment:
why do we not print the version if meta is `nil`? under what circumstances is it `nil`?

Reply:
oh... good catch. I think it stayed like this because in the beginning both things were printed on the same line and then we forgot to move the version out of the if...